NPE in BoatExample

NPE seen here: https://github.com/Backbase/backbase-openapi-tools/blob/66f80b063eb58e43fafda4c6b7c5227b419b8932/boat-scaffold/src/main/java/com/backbase/oss/codegen/doc/BoatExample.java#L42

```
Caused by: java.lang.NullPointerException
        at com.backbase.oss.codegen.doc.BoatExample.getPrettyPrintValue(BoatExample.java:42)
```

--- a/boat-engine/src/main/java/com/backbase/oss/boat/transformers/bundler/ExamplesProcessor.java
+++ b/boat-engine/src/main/java/com/backbase/oss/boat/transformers/bundler/ExamplesProcessor.java
@@ -253,7 +253,7 @@ public class ExamplesProcessor {
     private Object convertExampleContent(ExampleHolder exampleHolder, String refPath) {
         try {
             if (exampleHolder.getRef().endsWith("json") || refPath.endsWith("json")) {
-                return Json.mapper().readValue(exampleHolder.getContent(), Object.class);
+                return Json.mapper().readValue(exampleHolder.getContent(), JsonNode.class);
             }
             return exampleHolder.getContent();
         } catch (JsonProcessingException | RuntimeException e) {

--- a/boat-scaffold/src/main/java/com/backbase/oss/codegen/doc/BoatExample.java
+++ b/boat-scaffold/src/main/java/com/backbase/oss/codegen/doc/BoatExample.java
@@ -38,6 +38,8 @@ public class BoatExample {
         if (example.getValue() instanceof JsonNode
                 || example.getValue() instanceof HashMap) {
             return Json.pretty(example.getValue());
+        } else if (example.getValue() == null) {
+            return "";
         } else {
             return example.getValue().toString();
         }

